polybar-scripts/info-tmux-sessions/README.md

```diff
@@ -0,0 +1,21 @@
+# Script: info-tmux-sessions
+
+Reports the state of tmux sessions on the local machine.
+
+* with active sessions
+![info-tmux-sessions](screenshots/1.png)
+* with no server running
+![info-tmux-sessions](screenshots/2.png)
+
+## Dependencies
+
+* `tmux`
```

Comment:
we could delete this line, I guess its obviously that `tmux` is required

Reply:
I figured as much.  Should I remove the dependencies header too?

Reply:
Yes ..

Reply:
removed